rust/src/delta.rs

```diff
@@ -914,23 +924,36 @@ impl<'a> DeltaTransaction<'a> {
         }
     }
 
-    async fn try_commit(
+    async fn prepare_commit(
         &mut self,
         log_entry: &[u8],
+    ) -> Result<String, TransactionCommitAttemptError> {
+        let token = Uuid::new_v4().to_string();
+        let commit_path = self.delta_table.commit_path(&token);
+
+        self.delta_table
+            .storage
+            .put_obj(&commit_path, log_entry)
+            .await?;
+
+        Ok(commit_path)
+    }
+
+    async fn try_commit(
+        &mut self,
+        commit_file: &str,
```

Comment:
nitpick, would be good to use a different variable name to make the naming more consistent. Below we have log_path, which is also a path with string type, so we should use the same suffix here. For example, something like `tmp_log_path`.

Reply:
yeah, makes sense